src/openassetio-core/include/openassetio/managerApi/ManagerInterface.hpp

```diff
@@ -216,6 +217,171 @@ class OPENASSETIO_CORE_EXPORT ManagerInterface {
+     * methods:
+     * - @ref isEntityReferenceString
+     */
+    kEntityReferenceIdentification = 0,
+
+    /**
+     * Manager can provide a policy describing its behaviour with
+     * regard to specific entity types and relationships.
+     *
+     * Support for this capability is required by all managers. In
+     * situations where plugins are implemented as multiple component
+     * plugins (e.g. Python and C++) at least one of the component
+     * plugins must implement this capability.
+     *
+     * This capability means the manager implements the following
+     * methods:
+     * - @ref managementPolicy
+     */
+    kManagementPolicyQueries,
+
+    /**
+     * Manager makes use of the context to persist custom state for
+     * performance reasons or otherwise.
+     *
+     * Therefore, it is required that the host persist the context
+     * across related API calls, including across distributed processes.
+     *
+     * This capability means the manager implements the following
+     * methods:
+     * - @ref createState
+     * - @ref createChildState
+     * - @ref persistenceTokenForState
+     * - @ref stateFromPersistenceToken
+     *
+     * @see @ref stable_resolution
+     */
+    kStatefulContexts,
+    /**
+     * Manager customizes certain human-readable strings that the host
+     * might want to use in UI/messages.
+     *
+     * This capability means the manager implements the following
+     * methods:
+     * - @ref updateTerminology
+     */
+    kCustomTerminology,
+    /**
+     * Manager is capable of resolving @ref entity_reference into
+     * the data for one or more @ref trait "traits."
+     *
+     * This capability means the
+     * manager implements the following methods:
+     * - @ref resolve
+     */
+    kResolution,
+    /**
+     * Manager allows the host to create or update an @ref entity
+     * within the @ref asset_management_system
+     *
+     * This capability means the
+     * manager implements the following methods:
+     * - @ref preflight
+     * - @ref register_
+     */
+    kPublishing,
+    /**
+     * Manager is capable of querying entity references that are related
+     * to the input references by the relationship defined by a set of
+     * traits and their properties.
+     *
+     * This capability means the manager implements the following methods:
+     * - @ref getWithRelationship
+     * - @ref getWithRelationships
+     * - @ref getWithRelationshipPaged
+     * - @ref getWithRelationshipsPaged
+     */
+    kRelationshipQueries,
+    /**
+     * Manager is capable of confirming the existence of entities
+     *
+     * This capability means the manager implements the following
+     * methods:
+     * - @ref entityExists
+     */
+    kExistenceQueries,
+
+    /**
+     * Manager may be capable of a providing an @ref EntityReference
+     * considered to be a sensible default for each of the given entity
```

Comment:
[fix] As in other comment, "for each of the given" is misleading since the capability API doesn't take traits as arguments.